models/mednist_reg/configs/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "version": "0.0.1",
+    "changelog": {
+        "0.0.1": "Initial version"
+    },
+    "monai_version": "1.1.0",
+    "pytorch_version": "1.13.0a0+d0d6b1f",
```

Comment:
Thanks @wyli . Here could the pytorch version be a formal version, like `1.13.0`?

Reply:
this is autogenerated by https://github.com/Project-MONAI/MONAI/blob/5e6f105ff1d40266debfbd9774a9a0bdbe4d817a/monai/bundle/utils.py#L35

do you think we should add a split there?
```
>>> "1.13.0a0+d0d6b1f".split('a')[0]
'1.13.0'
```
we'll always have this kind of pytorch versioning in our docker image.